core/db_models/EEM_Ticket.model.php

```diff
@@ -277,11 +277,20 @@ protected function __construct($timezone)
      *
      * @return EE_Ticket[]
      * @throws EE_Error
+     * @throws ReflectionException
      */
-    public function get_all_default_tickets()
+    public function get_all_default_tickets(): array
     {
         /** @type EE_Ticket[] $tickets */
-        $tickets = $this->get_all([['TKT_is_default' => 1], 'order_by' => ['TKT_ID' => 'ASC']]);
+        $tickets = $this->get_all(
+            [
+                [
+                    'TKT_is_default' => 1,
+                    'TKT_visibility' => ['>', EEM_Ticket::TICKET_VISIBILITY_NONE_VALUE],
```

Comment:
now any positive value will indicate that the ticket has some level of visibility